webportal/lambda/terraform/configuration.tf

```
@@ -0,0 +1,33 @@
+provider "aws" {
+  version = "~> 2.0"
+  region  = "us-east-1"
+  profile = "sdc"
+}
+
+terraform {
+  required_version = "~> 0.12"
+
+  # All backend values must be supplied at init-time
+  backend "s3" {
+        key = "sdc-dot-webportal/webportal/lambda/terraform/terraform.tfstate"
+        region = "us-east-1"
+  }
+}
+
+
+data "aws_ssm_parameter" "account_number" {
+  name = "/common/secrets/account_number"
+}
```

```suggestion
data "aws_caller_identity" "current" {}

locals {
  account_id  = data.aws_caller_identity.current.account_id
  }
```
Not sure it's worth changing now because you would need to change `aws_ssm_parameter.account_number.value` for `local.account_id`. 

Seems like it's sixes, but this is the way I do it. However, we should come up with some standard ways of getting this kind of data. ¯\_(ツ)_/¯

---

updated - with minor nuance: locals.account_id places into input_variables.tf